src/inifile.cr

```diff
@@ -1,4 +1,9 @@
 class IniFile
+  # Parses an INI string.
```

Comment:
"INI string" isn't a particularly well defined term, maybe go for something like "Parses INI-style configuration from the given string"?